### PR TITLE
[NETBEANS-1074] Module Review j2eeserver

### PR DESCRIPTION
**Pull request by thilina01**
added 2 templates and 23 graphic files to licenseinfo.xml

### CODE REVIEW

File: j2eeserver/licenseinfo.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/deployment/impl/server-string.dtd</file>
```
Comment (junichi11) [inline j2eeserver/licenseinfo.xml]: This should be used here:
https://github.com/apache/incubator-netbeans/blob/91d94516c76ab50e8b420f70633694c025ca516e/platform/openide.util/src/org/openide/xml/XMLUtil.java#L179-L181

Comment (matthiasblaesing) [reply j2eeserver/licenseinfo.xml]: Same comment as in #722 applies here (both DTDs can hold the ALv2 header).

**Comment by thilina01** [comment]
@junichi11 @matthiasblaesing Please review the changes. Thanks. :)

**Comment by matthiasblaesing** [comment]
Thank you!

**Pull request merged**